doc/deoplete-jedi.txt

```diff
@@ -100,4 +100,4 @@ Once you have created them, add the following to your vimrc file: >
     let g:python_host_prog = '/full/path/to/neovim2/bin/python'
     let g:python3_host_prog = '/full/path/to/neovim3/bin/python'
 
-Note: deoplete only requires Python 3. See |nvim-python-quickstart|.
+Note: deoplete only requires Python 3. See h: nvim-python-quickstart.
```

Comment:
That does not work for me.
It seems to be `provider-python` in current Neovim - changed in https://github.com/neovim/neovim/commit/37f560ae.

So it should be `|provider-python|` probably (assuming that 0.2.0 is released soonish).